test/jest/acceptance/snyk-test/all-projects.spec.ts

```diff
@@ -145,9 +145,11 @@ describe('snyk test --all-projects (mocked server only)', () => {
     );
 
     expect(code).toEqual(1);
-    const req = server.popRequest();
-    expect(req.query.ignorePolicy).toBeTruthy(); // should request to ignore the policy
+    const requests = server.getRequests().filter((req: any) => {
+      if (req.query.ignorePolicy) return req.query.ignorePolicy;
```

Comment:
Suggestion: The statement in `filter()` should return a truthy or falsy value (see for example [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter#callbackfn)). This means that you can make all the statements simpler.

```suggestion
      return (req.query.ignorePolicy !== undefined);
```

Reply:
I refactored this and the other instances.